Shared/Samples/Show geodesic sector and ellipse/ShowGeodesicSectorAndEllipseView.swift

```diff
@@ -0,0 +1,385 @@
+        /// Changes the direction of the sector.
+        @Published var startDirection: Double = 45
+        
+        func refreshSector() {
+            guard let center = center else { return }
+            updateSector(center: center)
+        }
+        
+        private func updateSector(center: Point) {
+            ellipseGraphicOverlay.removeAllGraphics()
+            sectorGraphicOverlay.removeAllGraphics()
+            updateEllipse(center: center)
+            setupSector(center: center, geometryType: geometryType)
+        }
+        
+        private func setupSector(center: Point, geometryType: GeometryType) {
+            switch geometryType {
+            case .point:
+                // Generate sector as a multipoint (symbols).
+                var parameters = GeodesicSectorParameters<Multipoint>()
+                fillSectorParams(&parameters, center: center)
+                if let geometry = GeometryEngine.geodesicSector(parameters: parameters) {
+                    addSectorGraphic(
+                        geometry: geometry,
+                        symbol: SimpleMarkerSymbol(
+                            style: .circle,
+                            color: .green,
+                            size: 2
+                        )
+                    )
+                }
+            case .polyline:
+                // Generate sector as a polyline (outlined arc).
+                var parameters = GeodesicSectorParameters<Polyline>()
+                fillSectorParams(&parameters, center: center)
+                if let geometry = GeometryEngine.geodesicSector(parameters: parameters) {
+                    addSectorGraphic(
+                        geometry: geometry,
+                        symbol: SimpleLineSymbol(
+                            style: .solid,
+                            color: .green,
+                            width: 2
+                        )
+                    )
+                }
+            case .polygon:
+                // Generate sector as a filled polygon.
+                var parameters = GeodesicSectorParameters<ArcGIS.Polygon>()
+                fillSectorParams(&parameters, center: center)
+                if let geometry = GeometryEngine.geodesicSector(parameters: parameters) {
+                    addSectorGraphic(
+                        geometry: geometry,
+                        symbol: SimpleFillSymbol(
+                            style: .solid,
+                            color: .green
+                        )
+                    )
+                }
+            }
+        }
+        
+        /// Populates a `GeodesicSectorParameters<T>` instance with current user-defined values.
+        /// - Parameter parameters: A reference to the parameter struct that will be filled.
+        /// - Parameter center: The center point for the sector/ellipse.
+        private func fillSectorParams<T>(_ parameters: inout GeodesicSectorParameters<T>, center: Point) {
```

Comment:
Change:
 `private func fillSectorParams<T>(_ parameters: inout GeodesicSectorParameters<T>, center: Point)` 

to
 `private func fillSectorParameters<T>(_ parameters: inout GeodesicSectorParameters<T>, center: Point)`